libsession/src/main/java/org/session/libsession/messaging/sending_receiving/pollers/OpenGroupPollerV2.kt

```diff
@@ -103,4 +104,16 @@ class OpenGroupPollerV2(private val server: String, private val executorService:
             storage.setLastDeletionServerID(room, server, latestMax)
         }
     }
+
+    private fun downloadGroupAvatarIfNeeded(room: String) {
+        val openGroupId = "$server.$room"
+        val storage = MessagingModuleConfiguration.shared.storage
+        if (storage.getGroupAvatarDownloadJob(openGroupId) != null) return
```

Comment:
This may be the thing affected by that JobQueue overriding the ID. Would be interesting to see if there are multiple queued up on polling, but we may be able to remove the overwriting of the Job ID as it was primarily used for job receive IDs when they were not batched. Or even checking if it is null and only changing it if it is null

Reply:
Cool, will rethink this

Reply:
I tested only overriding null `id`s in the JobQueue and and it seemed to affect messaging jobs 🤔